nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -228,6 +228,14 @@ in
         '';
       };
 
+      banner = mkOption {
+        type = types.lines;
+        default = "";
```

Comment:
I think `nullOr lines` with `default = null` would be better so people can set empty banners too.

Reply:
Looking at sshd_config: ```
       Banner The contents of the specified file are sent to the remote user before authentication is allowed.  If the argument is none then no banner is displayed.  By default, no banner
              is displayed.
```
So I would say we should instead always write "Banner" to the configuration and just default it to "none" instead ?

Reply:
I see, yeah using `types.lines` with a default of `"none"` sounds reasonable